### ISSUE TITLE
[Bug] Command line arguments needs to be updated for the SDK Python testcases in UI

### ISSUE THREAD

**Issue by Saravana-kr22** [project-chip/certification-tool#125]
### Describe the bug

Command line arguments are missing in the UI SDK Python testcases. This cause the failure the testcasses

For ex: The TC-ACE-1.4 testcase is failed with the error `PIXIT.ACE.APPENDPOINT must be included on the command line in the --int-arg flag as PIXIT.ACE.APPENDPOINT:<endpoint>` this due to the missing argument  --int-arg PIXIT.ACE.APPENDPOINT::1

log: [tc-ace-1-4.log](https://github.com/project-chip/certification-tool/files/13674666/tc-ace-1-4.log)

For the below mentioned test cases are failed due to the missing command line arguments in UI. Hence the python command needs to be updated.

- TC_ACE_1_4 -  `python3 TC_ACE_1_4.py --discriminator 3840 --passcode 20202021 --commissioning-method on-network --storage-path admin_storage.json  --int-arg PIXIT.ACE.APPENDPOINT:1 PIXIT.ACE.APPDEVTYPEID:256 --string-arg PIXIT.ACE.APPCLUSTER:OnOff PIXIT.ACE.APPATTRIBUTE:OnOff  --paa-trust-store-path /credentials/development/paa-root-certs/`
- TC_OPSTATE_2_1 -  `python3  TC_OPSTATE_2_1.py --commissioning-method on-network --discriminator 3840 --passcode 20202021 --storage-path admin_storage.json --PICS ci-pics-values --int-arg PIXIT_ENDPOINT:1 PIXIT.OPSTATE.ErrorEventGen:1 --paa-trust-store-path /credentials/development/paa-root-certs/`
- TC_OPSTATE_2_3 -  `python3  TC_OPSTATE_2_3.py --commissioning-method on-network --discriminator 3840 --passcode 20202021 --storage-path admin_storage.json --PICS ci-pics-values --int-arg PIXIT_ENDPOINT:1 PIXIT.OPSTATE.ErrorEventGen:1 --paa-trust-store-path /credentials/development/paa-root-certs/`
- TC_RVCCLEANM_1_2 - `python3 TC_RVCCLEANM_1_2.py --commissioning-method on-network --discriminator 3840 --passcode 20202021 --storage-path admin_storage.json --PICS ci-pics-values --int-arg PIXIT_ENDPOINT:1 --paa-trust-store-path /credentials/development/paa-root-certs/`
- TC_RVCCLEANM_2_1 - `python3 TC_RVCCLEANM_2_1.py --commissioning-method on-network --discriminator 3840 --passcode 20202021 --storage-path admin_storage.json --PICS ci-pics-values  --int-arg PIXIT_ENDPOINT:1 PIXIT.RVCCLEANM.MODE_CHANGE_OK:2 PIXIT.RVCCLEANM.MODE_CHANGE_FAIL:0 --paa-trust-store-path /credentials/development/paa-root-certs/`
- TC_RVCCLEANM_3_2 - `python3 TC_RVCCLEANM_3_2.py --commissioning-method on-network --discriminator 3840 --passcode 20202021 --storage-path admin_storage.json --PICS ci-pics-values --int-arg PIXIT_ENDPOINT:1 --paa-trust-store-path /credentials/development/paa-root-certs/`
- TC_RVCOPSTATE_2_1 - `python3 TC_RVCOPSTATE_2_1.py --commissioning-method on-network --discriminator 3840 --passcode 20202021 --storage-path admin_storage.json --PICS ci-pics-values --int-arg PIXIT_ENDPOINT:1 --paa-trust-store-path /credentials/development/paa-root-certs/`
- TC_RVCOPSTATE_2_3 -` python3 TC_RVCOPSTATE_2_3.py --commissioning-method on-network --discriminator 3840 --passcode 20202021 --storage-path admin_storage.json --PICS ci-pics-values --int-arg PIXIT_ENDPOINT:1 --paa-trust-store-path /credentials/development/paa-root-certs/`
- TC_RVCRUNM_1_2 - `python3 TC_RVCRUNM_1_2.py --commissioning-method on-network --discriminator 3840 --passcode 20202021 --storage-path admin_storage.json --PICS ci-pics-values --int-arg PIXIT_ENDPOINT:1 --paa-trust-store-path /credentials/development/paa-root-certs/
`- TC_RVCRUNM_2_1 - python3 TC_RVCRUNM_2_1.py --commissioning-method on-network --discriminator 3840 --passcode 20202021 --storage-path admin_storage.json --PICS ci-pics-values --int-arg PIXIT_ENDPOINT:1 PIXIT.RVCRUNM.MODE_CHANGE_OK:2 PIXIT.RVCRUNM.MODE_CHANGE_FAIL:0  --paa-trust-store-path /credentials/development/paa-root-certs/
- TC_RVCRUNM_3_2 - `python3 TC_RVCRUNM_3_2.py --commissioning-method on-network --discriminator 3840 --passcode 20202021 --storage-path admin_storage.json --PICS ci-pics-values --int-arg PIXIT_ENDPOINT:1 PIXIT_MODEFAIL:4 --paa-trust-store-path /credentials/development/paa-root-certs/`

**Comment by rquidute** [comment]
Hi @Saravana-kr22 could you please add the missing arguments in Project Config,following the example bellow:
```
 "test_parameters": {
      "paa-trust-store-path": "/paa-root-certs",
      "storage-path": "/root/admin_storage.json",
      "int-arg": "PIXIT.ACE.APPENDPOINT:1 PIXIT.ACE.APPDEVTYPEID:256",
      "string-arg": "PIXIT.ACE.APPCLUSTER:OnOff PIXIT.ACE.APPATTRIBUTE:OnOff"
    }
```

<img width="894" alt="Screenshot 2023-12-14 at 12 35 31" src="https://github.com/project-chip/certification-tool/assets/116586593/712ee4f3-f811-44f2-8068-89164b598f75">



- TC_ACE_1.4 passed after adding these configurations.

<img width="1306" alt="Screenshot 2023-12-14 at 12 32 44" src="https://github.com/project-chip/certification-tool/assets/116586593/8b909579-3ab1-400b-9727-8cb7b006cfee">

**Comment by Saravana-kr22** [comment]
Hi @rquidute I have tried with the above test-parameters but the arguments werenot passed to the script with the message like Argument paa-trust-store-path is not valid and the test cases is failed due to this.

![image](https://github.com/project-chip/certification-tool/assets/104007654/38ae8318-c2a3-4c46-80ce-6ed5978942f2)

This is the project config I used for execution:

` {
  "config": {
    "network": {
      "wifi": {
        "ssid": "testharness",
        "password": "wifi-password"
      },
      "thread": {
        "rcp_serial_path": "/dev/ttyACM0",
        "rcp_baudrate": 115200,
        "on_mesh_prefix": "fd11:22::/64",
        "network_interface": "eth0",
        "dataset": {
          "channel": "15",
          "panid": "0x1234",
          "extpanid": "1111111122222222",
          "networkkey": "00112233445566778899aabbccddeeff",
          "networkname": "DEMO"
        },
        "otbr_docker_image": null
      }
    },
    "dut_config": {
      "discriminator": "3840",
      "setup_code": "20202021",
      "pairing_mode": "onnetwork",
      "chip_tool_timeout": null,
      "chip_tool_use_paa_certs": false
    },
    "test_parameters": {
      "paa-trust-store-path": "/paa-root-certs/",
      "storage-path": "/root/admin_storage.json",
      "int-arg": "PIXIT.ACE.APPENDPOINT:1 PIXIT.ACE.APPDEVTYPEID:256",
      "string-arg": "PIXIT.ACE.APPCLUSTER:OnOff PIXIT.ACE.APPATTRIBUTE:OnOff"
    }
  },
  "pics": {
    "clusters": {}
  }
} `

**Comment by rquidute** [comment]
Oh I see the problem @Saravana-kr22 . This will be fixed in next release. But you can still run in the current version if you change the test arguments by replacing the `-` to `_` for example: from `paa-trust-store-path` to `paa_trust_store_path`, `storage-path` to `storage_path` and so on.
In the next release you should use argument in the way it is expected by the python test, for example: paa-trust-store-path and storage-path.

**Comment by hiltonlima** [comment]
Fixed test_parameters section
```
 "test_parameters": {
      "paa_trust_store_path": "/paa-root-certs/",
      "storage_path": "/root/admin_storage.json",
      "int_arg": "PIXIT.ACE.APPENDPOINT:1 PIXIT.ACE.APPDEVTYPEID:256",
      "string_arg": "PIXIT.ACE.APPCLUSTER:OnOff PIXIT.ACE.APPATTRIBUTE:OnOff"
    }
```

**Comment by Saravana-kr22** [comment]
This issue has been resolved in latest image `Version: 2.10-beta2+spring2024
`
